src/trello2md.py

```diff
@@ -141,6 +144,10 @@ def main():
 
     args = parser.parse_args()
 
+    if args.card_links:
+      print('Option --card-links is currently unimplemented')
+      exit(1)
```

Comment:
I wouldn't like the program to exit just because an unimplemented functionality was used.  Let's keep it to a warning.